static/templates/settings/account-settings.handlebars

```diff
@@ -212,13 +212,15 @@
         <div id="show_api_key_box">
             <p>{{t "Your API key:" }}</p>
             <p><b><span id="api_key_value"></span></b></p>
-            <button type="submit" class="button white rounded regenerate_api_key">
-                {{t "Generate new API key" }}
-            </button>
-            <a id="download_zuliprc" download="{{zuliprc}}" class="button sea-green">
-                {{t "Download .zuliprc" }}
-            </a>
-            <div id="user_api_key_error text-error">
+            <div class="flexbox">
+                <button type="submit" class="button white rounded regenerate_api_key">
+                    {{t "Generate new API key" }}
+                </button>
+                <a id="download_zuliprc" download="{{zuliprc}}" class="button sea-green">
+                    {{t "Download .zuliprc" }}
+                </a>
+                <div id="user_api_key_error text-error">
```

Comment:
I don't think this is valid HTML (you can't have two IDs for an element). Shouldn't text-error be a class?

Reply:
Once you resolve the `text-error` issue, you'll find that the error is being incorrectly displayed inline with the buttons (it should be on a separate line under). Can you correct this regression?

![screenshot at oct 13 16-40-58](https://user-images.githubusercontent.com/15116870/46911102-f7dcdb80-cf06-11e8-94cf-a2e880dbaa61.png)

Reply:
@synicalsyntax 

1) Changed text-error as class.

2)  text-error issue resolved.

![error](https://user-images.githubusercontent.com/31280303/46917185-802faf00-cfe1-11e8-8908-85bc83dd76d0.png)